components/leaflet-maps/NormalMap.vue

```diff
@@ -142,22 +152,29 @@
        let locationsWithValue = []
 
        this.locationOptions.map(option => {
-         if (witch[option].location !== "") {
+         if (witch[option].locations.length !== 0) {
            locationsWithValue.push(option);
          }
        })
 
        return locationsWithValue
      },
+     getStandardsWithValue: function (witch) {
```

Comment:
Same as the previous comment. Probably better to name getStandardAttributesWithValues.

Thanks